### ISSUE TITLE
`--container-builder` doesn't respect --non-interactive

### ISSUE THREAD

**Issue by justinyoo** [prom3theu5/aspirational-manifests#208]
### 🔥 Bug Description

```bash
aspirate --version

8.0.1+5575b4754137411aea2977ae5519429d4745f3df
```

![image](https://github.com/prom3theu5/aspirational-manifests/assets/1538528/de93647a-4e3e-4540-85eb-531f3d3673a7)

I tried the following command:

```bash
# Case 1
aspirate init -cr <container-registry-url> -ct latest --non-interactive
```
And I got this prompt, which shouldn't be displaying:

![image](https://github.com/prom3theu5/aspirational-manifests/assets/1538528/f34c1662-3a03-4a8e-baa7-f7b0571aa755)

Then, I tried the following command:

```bash
# Case 2
aspirate init -cr <container-registry-url> -ct latest --container-builder Docker --non-interactive
```

And I got this error, which should take the parameter value case insensitive:

![image](https://github.com/prom3theu5/aspirational-manifests/assets/1538528/26002828-f519-4dcb-a9e8-ea7691445c39)

Then, I tried the following command:

```bash
# Case 3
aspirate init -cr <container-registry-url> -ct latest --container-builder docker --non-interactive
```

And I got this prompt, which shouldn't be displaying:

![image](https://github.com/prom3theu5/aspirational-manifests/assets/1538528/b36faee2-ea3e-49ec-b695-3f1b60290d54)

All of the cases above use the parameter `--non-interactive`, which should assume the default value.

### 🔍 Steps to Reproduce the Bug

Above 👆

### 🧯 Possible Solution

- If no `--container-builder` parameter provided, it should default to `docker`.
- If `Docker` is provided for the `--container-builder`, it should be accepted instead of throwing the exception.

**Comment by prom3theu5** [comment]
I admit the init command hasn't seen much love over the past few months, as most of its functionality is now usable from all the commands directly. I have a bug fix to perform tonight / this weekend so I will address this the same time - Thanks for reporting

**Comment by prom3theu5** [comment]
Fixed in #211 
Going out now in 8.0.2
